test/test_utils.h

```diff
@@ -463,6 +471,47 @@ namespace test_utils {
 		std::unique_ptr<detail::buffer_manager> bm;
 	};
 
+#ifdef _WIN32
+
+	class affinity_fixture {
```

Comment:
I vote for moving this to `runtime_tests.cc` since that's the only place where the fixture is used.

Also IMO, `restore_process_affinity_fixture` or `reset_process_affinity_fixture` would be more descriptive names for this class.

Also, pedantic: copy ctors/`operator=`s should be deleted.

Reply:
> I vote for moving this to runtime_tests.cc since that's the only place where the fixture is used.

I just put this fixture with the other fixtures, I thought that's where they go. But I agree that it makes more sense in `runtime_tests.cc`, so I moved it there.

Also renamed it and deleted the unused ctors/operators.